src/stories/Jobs/CanUpdateJob/tests.spec.js

```diff
@@ -0,0 +1,114 @@
+const debugLogger = requireUtil("debugLogger");
+const knex = requireKnex();
+const randomUser = requireUtil("randomUser");
+const contextClassRef = requireUtil("contextHelper");
+const JobsRepo = requireRepo("jobs");
+
+describe("Test Handler Jobs/UserCanUpdateJob", () => {
+  it("dummy_story_which_will_pass", async () => {
```

Comment:
this story would be rejected, let me know if its WIP.

Reply:
yes, and I haven't started the jobs table yet. I'll change the code accordingly once I will work on jobs table